src/core/src/op/read_value.cpp

```diff
@@ -179,6 +179,10 @@ bool ReadValue::has_evaluate() const {
 bool ReadValue::constant_fold(OutputVector& output_values, const OutputVector& inputs_values) {
```

Comment:
Can be removed as Base implementation will return false.

Reply:
done